### PR TITLE
doc/ipad.md: add install pwa

### PR DESCRIPTION
**Pull request by jsjoeio**
Update the iPad docs with instructions on how to install the PWA.

See comment: https://github.com/cdr/code-server/issues/2586#issuecomment-767315912

### CODE REVIEW

File: doc/ipad.md [hunk 1]
```diff
@@ -28,6 +29,16 @@
...
 
+## How to install PWA
+
+To install the code-server PWA, follow these steps:
+
+1. Open code-server in Safari
+2. Click the Share icon
+3. Click "Add to Home Screen"
+
+Now when you open code-server from the home screen, you will be using the PWA. The advantages of this are more screen real estate and access to top-level keyboard shortcuts because it's running like an app.
```
Comment (nhooyr) [inline doc/ipad.md:40]: Lets add an example of such a shortcut and hard wrap this line.

File: doc/ipad.md [hunk 2]
```diff
@@ -28,6 +30,31 @@
...
+
+To install the code-server PWA, follow these steps:
+
+1. Open code-server in Safari
+2. Click the Share icon
+3. Click "Add to Home Screen"
+
+Now when you open code-server from the home screen, you will be using the PWA.
+The advantages of this are more screen real estate and access to top-level keyboard shortcuts because it's running like an app.
+An example shortcut is the `cmd+w` to close an active file in the workbench. You can add this to your `keybindings.json` by doing the following:
```
Comment (jsjoeio) [inline doc/ipad.md:43]: Is this okay? If not, we should update the `yarn fmt` script to hard-wrap to the line length we want.

Comment (oxy) [reply doc/ipad.md:42]: As long as CI does not complain, I think it's fine in my book!

**Review by oxy — APPROVED** [review]
Looks good!

**Pull request merged**